src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+struct gss_name_t_desc_struct;
+typedef struct gss_name_t_desc_struct* gss_name_t;
```

Comment:
> Why are thee typedef'd? That just adds unnecessary indirection and complexity in the code. Please don't do this

I had to typedef these to get the forward declarations.  Is there a way to fwd declare gss_name_t without introducing the typedefs?